src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+
+import { IconType } from '../../types/IconType';
+import './Icon.scss';
+
+type Props = {
+  type: IconType;
+  addClassName?: string;
+};
+
+export const Icon: React.FC<Props> = ({ type, addClassName }) => {
+  return (
+    <span
+      className={classNames(
+        'icon',
+        type,
+        addClassName,
+      )}
+    />
+  );
+};
```

Comment:
Let's take one approach and return this component in the same way as you do in the components: Loader, ErrorNotification, Header and so on:

```suggestion
export const Icon: React.FC<Props> = ({ type, addClassName }) => (
  <span
    className={classNames(
      'icon',
      type,
      addClassName,
    )}
  />
);
```

Reply:
No problem😉
Done ✅